telegram/ext/_application.py

```diff
@@ -1132,7 +1139,11 @@ async def process_update(self, update: object) -> None:
                             and self.bot.defaults
                             and not self.bot.defaults.block
                         ):
-                            self.create_task(coroutine, update=update)
+                            self.create_task(
+                                coroutine,
+                                update=update,
+                                name=f"Application:process_update_non_blocking:{handler}",
```

Comment:
Handlers usually don't have readible representations. Is that worth adding? Bot.id (also for the rest) and update ID?

Reply:
I tried to include a dynamic part to every task name (so that it's unique and easy to distinguish while inspecting all running tasks). 

Can't use update id for the same reason above.

> Handlers usually don't have readible representations

Maybe we should make them readable? Should I open a new issue?

Reply:
Fair point. Yes, let's open an issue